### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by gem-neo4j**
I found some issues with the formatting and links, so fixed them up!

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: modules/ROOT/pages/functions/scalar.adoc [hunk 1]
```diff
@@ -1499,7 +1499,7 @@ With this in mind, the below list contains all supported types (as of Neo4j 5.13
 
 This should be taken into account when relying on the output of the `valueType()` function.
 
-See the xref::syntax/expressions.adoc#type-predicate-expressions[type predicate expression] for an alternative way of testing type values.
+See the xref::values-and-types/type-predicate.adoc[type predicate expression] for an alternative way of testing type values.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/scalar.adoc:1502]: This should be plural as well, right? ("type predicate expressions")

**Review by JPryce-Aklundh — APPROVED** [review]
Thanks, @gem-neo4j!

File: modules/ROOT/pages/functions/scalar.adoc (hunk 1 above)
Comment (gem-neo4j) [reply modules/ROOT/pages/functions/scalar.adoc:1502]: Well it is just one expression though 🤔 Would we say "see the Count Expression" or "see the Count Expressions"? :)

**Pull request merged**